### DISCUSSION TITLE
What happened to the content that was in the 'images' and 'www' folders?

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [ImageMagick/ImageMagick#4055] [Help] [answered]
Greetings.  Thank you for ImageMagick.

I used ImageMagick a few years ago.  The version I used was contained in `ImageMagick-7.0.9-8-portable-Q16-x64.zip`.  That zip file included two folders that I do not see in the current portable release:
1. `images` (with a subfolder named `patterns`)
2. `www`

It looks like these folders contained a useful tutorial and/or perhaps a "help" website.  Is this content now available elsewhere?

Did those folders contain anything else worth saving?  I noticed that the `images` folder had a `patterns` subfolder that contained a bunch of useful pattern images.  Are they now available elsewhere?

**Comment by dlemstra** [comment, marked as answer]
The content of the `www` folder is our website that can be found here: https://imagemagick.org/.  The `images` folder (and the `www` folder) can be found in our GitHub repository: https://github.com/ImageMagick/ImageMagick/tree/main.

**Reply by Gitoffthelawn** [reply]
Thank you so very much! <3

In one of the folders (I think it was the `images` folder there was a before and after photograph of a mountain scene.  Do you happen to know where that is on the website that is located so I can learn about what settings were used to generate those effects?

**Reply by dlemstra** [reply]
Are you looking for this page: https://imagemagick.org/script/clahe.php?

**Reply by Gitoffthelawn** [reply]
@dlemstra Thank you! Thank you! Thank you!  That's exactly the page for which I was looking!  Thank you!  <3
